packages/devtools/src/transactions/format.ts

```diff
@@ -3,9 +3,12 @@ import { formatEid } from '@/omnigraph/format'
 
 export const formatOmniTransaction = (
     transaction: OmniTransaction
-): Record<string, string | number | bigint | undefined> => ({
+): Record<string, string | number | bigint | undefined | null> => ({
     Endpoint: formatEid(transaction.point.eid),
     OmniAddress: transaction.point.address,
+    OmniContract: transaction.point.contractName,
+    'Function Name': transaction.functionName,
+    'Function Arguments': transaction.functionArgs,
```

Comment:
Rather than growing the `OmniTransaction` object, I'd create a new `OmniTransactionMetadata` interface like so:

```typescript
interface OmniTransactionMetadata {
  functionName?: string
  functionArgs?: unknown[]
}
```

Then you add a `metadata?: OmniTransactionMetadata` property to `OmniTransaction`. This way we keep the meta properties isolated.

Reply:
Ah great idea! On it 🫡